### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by palmfjord**
#88 is merged and all content is published, lets ship this puppy!

### CODE REVIEW

File: src/server/middlewares/states.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Middleware } from 'koa'
+import { getBlogPosts } from '../utils/storyblok'
+import { getCachedTeamtailorUsers } from '../utils/teamtailor'
+
+export const addTeamtailorUsersToState: Middleware = async (ctx, next) => {
```
Comment (hedvigoscar) [inline src/server/middlewares/states.ts]: I like this! If I understand correctly it means that we only load the TeamTailor users when accessing a page that requires them, right? 🙂

File: src/utils/meta.tsx [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
   <>
-    <title>{story.name}</title>
+    <title>{title ? title : story && story.name}</title>
     <link
       rel="canonical"
-      href={`${getPublicHost()}/${story.full_slug.replace(/\/?home$/, '')}`}
+      href={`${getPublicHost()}/${fullSlug || getFullSlugFromStory(story)}`}
     />
-    {story.content.robots && (
+    {story && story.content.robots && (
```
Comment (hedvigoscar) [inline src/utils/meta.tsx]: Doesn't this code hurt you a little? :wink: I feel like there's probably a good ramda-style function for doing this 🙂

**Comment by hedvigoscar** [comment]
Ok! Will it still be served as quickly as it would be from our CDN? 🙂

**Comment by hedvigoscar** [comment]
Maybe it's a setting for the CF we have in front of `web-next`? 🙂

**Comment by palmfjord** [comment]
maybe there'll be a couple of milliseconds latency more, but it's still behind cloudfront and has the same caching settings so i don't think it's nociable

**Comment by palmfjord** [comment]
@hedvigoscar how do you mean? :)

**Pull request merged**